File: Python/URI/1074.py
casos = int(input())
i = 0
for i in range(casos):

    num = int(input())
    if(num == 0):
        print("NULL")
    if(num % 2 != 0 and num < 0):
        print("ODD NEGATIVE")
    if(num % 2 == 0 and num < 0):
        print("EVEN NEGATIVE")
    if(num % 2 != 0 and num > 0):
        print("ODD POSITIVE")
    if(num % 2 == 0 and num > 0):
        print("EVEN POSITIVE")

File: JS/URI/1074.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var caso = parseInt(lines.shift());

for (; caso > 0; caso--) {

   var num = parseInt(lines.shift());
    if (num === 0)
        console.log("NULL");
    if (num % 2 !== 0 && num < 0)
        console.log("ODD NEGATIVE");
    if (num % 2 === 0 && num < 0)
        console.log("EVEN NEGATIVE");
    if (num % 2 !== 0 && num > 0)
        console.log("ODD POSITIVE");
    if (num % 2 === 0 && num > 0)
        console.log("EVEN POSITIVE");
}